parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         - "--tls-private-key-file=/etc/kubernetes/certs/apiserver.key"
         - "--client-ca-file=/etc/kubernetes/certs/ca.crt"
         - "--service-account-key-file=/etc/kubernetes/certs/apiserver.key"
+        - "--oidc-client-id=<oidcClientId>"
+        - "--oidc-issuer-url=<oidcIssuerUrl>"
+        - "--oidc-username-claim=oid"
```

Comment:
It looks like the way that these three `--oidc*` options are defined, and populated below in `kubernetesmastercustomdata.yml`, doesn't accommodate an api model without an AAD profile. Am I incorrect? We want to ensure that this additional configuration has no effect on existing users/api models.